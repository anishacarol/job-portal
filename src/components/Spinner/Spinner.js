import React from "react";
import * as Styled from "./Spinner.styles";

const Spinner = () => (
  <Styled.Loader>
    <div className="loader"></div>
  </Styled.Loader>
);

export default Spinner;
