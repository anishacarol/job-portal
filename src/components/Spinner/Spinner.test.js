import { render } from "@testing-library/react";
import React from "react";
import Spinner from "./Spinner";

describe("rendering", () => {
  test("renders Spinner correctly", () => {
    const { asFragment } = render(<Spinner />);

    expect(asFragment()).toMatchSnapshot();
  });
});
