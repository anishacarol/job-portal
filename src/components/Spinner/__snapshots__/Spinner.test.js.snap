// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders Spinner correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm hKwWCC"
  >
    <div
      class="loader"
    />
  </div>
</DocumentFragment>
`;
